README.md

```diff
@@ -100,6 +100,11 @@ and put it in the `ankisyncd` directory.
     addr = "http://127.0.0.1:27701/" # put your server address here
     os.environ["SYNC_ENDPOINT"] = addr + "sync/"
     os.environ["SYNC_ENDPOINT_MEDIA"] = addr + "msync/"
+#### anki 2.1.41-2.1.43
```

Comment:
So this seems like a fix for a known issue. Can you add a section called `Known Issues` and add a PR number to the issue.